diff --git a/rollup.config.js b/rollup.config.js
index 7a7a443..b47d2a2 100644
--- a/rollup.config.js
+++ b/rollup.config.js
@@ -1,19 +1,20 @@
-import babel from 'rollup-plugin-babel'
-import { terser } from 'rollup-plugin-terser'
-import commonjs from '@rollup/plugin-commonjs'
-import resolve from '@rollup/plugin-node-resolve'
+const babel = require('@rollup/plugin-babel');
+const { terser } = require('rollup-plugin-terser');
+const commonjs = require('@rollup/plugin-commonjs');
+const resolve = require('@rollup/plugin-node-resolve');
 
 const babelConfig = {
   babelrc: false,
   presets: [['@babel/preset-env', { targets: 'defaults, IE >= 10, Safari >= 5.1' }]]
 }
 
-export default [{
+module.exports = [{
   input: 'lib/browser.js',
-  output: { file: 'build/qrcode.js', format: 'iife', name: 'QRCode', exports: 'named' },
+  output: { file: 'build/qrcode.js', format: 'iife', name: 'QRCode', exports: 'named', globals: { dijkstrajs: 'dijkstrajs' } },
+  external: ['dijkstrajs'],
   plugins: [commonjs(), resolve(), babel(babelConfig), terser()]
 }, {
   input: 'helper/to-sjis-browser.js',
-  output: { file: 'build/qrcode.tosjis.js', format: 'iife', exports: 'none' },
+  output: { file: 'build/qrcode.tosjis.js', format: 'iife', exports: 'default' },
   plugins: [commonjs(), resolve(), babel(babelConfig), terser()]
 }]
